portality/scripts/manage_background_jobs.py

```diff
@@ -144,40 +170,311 @@ def query(self):
+
+
+def report(example_size=10):
+    from portality.bll.services.huey_job import huey_job_service
+    huey_rows = list(huey_job_service.find_all_huey_jobs())
+
+    scheduled = Counter(r.bgjob_action for r in huey_rows if r.is_scheduled)
+    unscheduled = Counter(r.bgjob_action for r in huey_rows if not r.is_scheduled)
+
+    print(title('# Huey jobs:'))
+    print(title(f'### Scheduled ({total_counter(scheduled)})'))
+    print_huey_jobs_counter(scheduled)
+    print()
+
+    print(title(f'### Unscheduled ({total_counter(unscheduled)})'))
+    print_huey_jobs_counter(unscheduled)
+    print()
+
+    print(title(f'### last {example_size} unscheduled jobs:'))
+    unscheduled_huey_jobs = sorted((r for r in huey_rows if not r.is_scheduled),
+                                   key=lambda x: x.schedule_time, reverse=True)
+    for r in unscheduled_huey_jobs[:example_size]:
+        print_huey_job_data(r)
+    print()
+
+    counter = Counter(r.bgjob_id for r in huey_rows if not r.is_scheduled)
+    counter = {k: v for k, v in counter.items() if v > 1}
+    if counter:
+        print(title(f'### Duplicated jobs ({total_counter(counter)})'))
+        for k, v in counter.items():
+            print(f'{k} {v}')
+
+    print_huey_jobs_counter(counter)
+
+    print(title('# DB records'))
+    print(title(f'### Processing'))
+    for j in find_bgjobs_by_status(constants.BGJOB_STATUS_PROCESSING):
+        print(job_to_str(j))
+    print()
+
+    bgjobs: List[BackgroundJob] = find_bgjobs_by_status(BGJOB_STATUS_QUEUED)
+    counter = Counter((j.action, j.status) for j in bgjobs)
+    print(title(f'### Queued ({sum(counter.values())})'))
+    for k, v in sorted(counter.items(), key=lambda x: x[0]):
+        print(f'{k[0]:30} {k[1]:10} {v}')
+    print()
+
+    print(title(f'### last {example_size} jobs'))
+    for j in sorted(bgjobs, key=lambda j: j.created_date, reverse=True)[:example_size]:
+        print(job_to_str(j))
+    print()
+
+    print(title('# Queued delta between DB and redis:'))
+    huey_only, db_only = find_huey_bgjob_delta(unscheduled_huey_jobs, bgjobs)
+    print_job_delta('### DB only', ((j.id, j.action) for j in db_only))
+    print_job_delta('### Redis only', ((i.bgjob_id, i.bgjob_action) for i in huey_only))
+
+
+def rm_all():
+    from portality.bll.services.huey_job import huey_job_service
+    if input(color_text.apply_color(
+            'WARNING: This will delete all jobs from redis and the database. Proceed? [y\\N] ',
+            background=color_text.Color.red)).lower() != 'y':
+        print('No action.')
+        return
+    from portality import models
+
+    print('Remove all jobs from DB')
+    models.BackgroundJob.delete_by_query(es_queries.query_all())
+
+    print('Remove all jobs from redis')
+    client = huey_job_service.create_redis_client()
+    client.delete('huey.redis.doajmainqueue')
+    client.delete('huey.redis.doajlongrunning')
```

Comment:
These two strings are also in the service layer in constant names, so I would use those instead https://github.com/DOAJ/doaj/pull/2327/files#diff-14da6aba45af431e9622b01afd9fb2c38ed0b392017624f559828871e9864b6aR41